CONTRIBUTING.md

```diff
@@ -104,6 +104,30 @@ a ticket number.
 * After feedback has been given we expect responses within two weeks. After two
   weeks we may close the pull request if it isn't showing any activity.
 
+## Revert Policy
+By running tests in advance and by engaging with peer review for prospective
+changes, your contributions have a high probability of becoming long lived
+parts of the the project. After being merged, the code will run through a
+series of testing pipelines on a large number of operating system
+environments. These pipelines can reveal incompatibilities that are difficult
+to detect in advance.
+
+If the code change results in a test failure, we will make our best effort to
```

Comment:
The regression discovery is the trigger for this situation. It is loosely coupled to the time of the merge depending on the status of the test pipelines and how quickly they get run after the merge. So, from the contributers standpoint, talking about a time duration based on the time of merge is more difficult to honor consistently.

Would it me more amenable to discuss an amount of time in terms of time from regression discovery?

Example
```
If the code change results in a test failure, we will make our best effort to
correct the error. If a fix cannot be determined and committed within 24 hours of its discovery,
the commit(s) responsible will be reverted.
```